src/oauth/auth.js

```diff
@@ -63,16 +63,18 @@ export default async function auth(
   tokens,
   callbackUrl,
   {accessType, forSignIn = false, forceLogin = false, screenName = ''} = {},
+  openUrl
```

Comment:
I think `openUrl` should be a property of `options` (`auth(tokens, callbackUrl, {openUrl})`).